### ISSUE TITLE
Create organization

### ISSUE THREAD

**Issue by dbosk** [repobee/repobee#1026]
It would be nice if RepoBee also could set up the organization with suitable settings (if any needed). One thing less to do in the GUI :-)

**Comment by slarse** [comment]
At some point, many years ago, I'm fairly certain there was a good reason for not doing that. But I don't remember what that reason was, and much may have changed since.

The one thing off the top of my head that was an inconvenience was that organizations on GitHub required a paid plan for private repositories, but that isn't the case anymore. There's still pricing for having "collaborators" on repositories, though, so I need to experiment a little bit with what's possible in the free tier.

Basically, if it can't be setup "perfectly" and you still need to do something in the web interface, I think it's better to do it all in the web interface. But I'll have a look :)

**Comment by dbosk** [comment]
Sounds reasonable. I was thinking about creating the organisation for a course in autumn and realized I don't know what settings I must adjust (on gits-15, for now).Maybe it's documented somewhere, that would be helpful, but then it felt like something that should be possible to automate. If it can be automated, it should be automated --- manual labour is for computers 😉Sent from a low-usability deviceOn 8 Apr 2022 19:30, Simon Larsén ***@***.***> wrote:
At some point, many years ago, I'm fairly certain there was a good reason for not doing that. But I don't remember what that reason was, and much may have changed since.
The one thing off the top of my head that was an inconvenience was that organizations on GitHub required a paid plan for private repositories, but that isn't the case anymore. There's still pricing for having "collaborators" on repositories, though, so I need to experiment a little bit with what's possible in the free tier.
Basically, if it can't be setup "perfectly" and you still need to do something in the web interface, I think it's better to do it all in the web interface. But I'll have a look :)

**Comment by slarse** [comment]
There are really two parts to it, the general platform quirks that are documented here: https://docs.repobee.org/en/stable/fundamentals.html#usage-with-different-platforms-github-github-enterprise-and-gitlab

And the instructions for creating an organization on GitHub that are documented here: https://docs.repobee.org/en/stable/getting_started.html#create-an-organization

**Comment by dbosk** [comment]
Thanks for the pointers to the documentation!

**Comment by dbosk** [comment]
Seems like there is not much setup that must be done, just that members privileges option.
